packages/block-library/src/post-author-biography/edit.js

```diff
@@ -10,40 +10,30 @@ import {
 	AlignmentControl,
 	BlockControls,
 	useBlockProps,
+	store as blockEditorStore,
 } from '@wordpress/block-editor';
 import { useSelect } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
-import { store as coreStore } from '@wordpress/core-data';
 
 function PostAuthorBiographyEdit( {
-	context: { postType, postId },
```

Comment:
This was returning undefined, so the logic after wasn't working at all. I'm not sure if it's ok to remove

Reply:
mmh, this might be working on the post editor but breaking on the site editor. Now I just checked and my change broke it in the post editor :DDD